cvat-core/src/server-proxy.ts

```diff
@@ -1817,7 +1634,7 @@ async function uploadAnnotations(
         }
     }
     try {
-        return await wait();
+        return rqId;
```

Comment:
1. Please, remove try catch from return value
2. Please, refactor if/else for cloud storage / not a cloud storage